wasm/python/itkwasm-elastix-wasi/itkwasm_elastix_wasi/read_parameter_files.py

```diff
@@ -51,7 +51,7 @@ def read_parameter_files(
     if len(parameter_files) > 0:
         args.append('--parameter-files')
         for value in parameter_files:
-            input_file = str(PurePosixPath(parameter_files))
+            input_file = str(PurePosixPath(value))
```

Comment:
This looks like an obvious bug fix 👍  However, it's not a bug fix commit, right? How could this code have passed the tests before?